### ISSUE TITLE
Search now requires ID

### ISSUE THREAD

**Issue by patvdleer** [thomaxxl/safrs#31]
Currently testing the latest release, https://github.com/thomaxxl/safrs/commit/8afc35d7c532eb480d70cec5d20b86e83e0832c5 but the search and re_search now requires an ID

![image](https://user-images.githubusercontent.com/1138136/54422181-2ce79700-470e-11e9-9d1e-6c9edd744069.png)

Before extending on the `SAFRSBase` class I do the following;
```python
from safrs import SAFRSBase
from safrs.api_methods import search, re_search

SAFRSBase.search = search
SAFRSBase.re_search = re_search
```

**Comment by thomaxxl** [comment]
arg, thanks for letting me know.
I didn't have time to properly test it and I'm implementing new testing anyway.
Will update asap.

**Comment by thomaxxl** [comment]
I resolved this but I still have to figure out exactly why this error occured in the first place.
I'm doing some low-level inspection of the methods but there seems to be a difference between 
`__dict__.get` and `getattr` that I don't fully understand. Probably because of the decorators that were applied. I'll look into it.

**Comment by thomaxxl** [comment]
fixed in v2.1.0
